docs/reference/api/helpers.rst

```diff
@@ -11,16 +11,16 @@ hipSOLVER helper functions
 These are helper functions that control aspects of the hipSOLVER library. They are divided
 into the following categories:
 
-* :ref:`initialize` functions. Used to initialize and cleanup the library handle.
-* :ref:`stream` functions. Provide functionality to manipulate streams.
-* :ref:`determinism` functions. Provide functionality to manipulate function determinism.
-* :ref:`gesvdj_info` functions. Provide functionality to manipulate gesvdj parameters.
-* :ref:`syevj_info` functions. Provide functionality to manipulate syevj parameters.
+* :ref:`initialize` functions initialize and cleanup the library handle.
+* :ref:`stream` functions manipulate streams.
+* :ref:`determinism` functions manipulate function determinism.
+* :ref:`gesvdj_info` functions manipulate gesvdj parameters.
+* :ref:`syevj_info` functions manipulate syevj parameters.
```

Comment:
It would be nice if these matched the corresponding descriptions in dense-api/helpers.rst